Gemfile

```diff
@@ -3,10 +3,18 @@ source 'https://rubygems.org'
 ruby '2.3.1'
 
 gem 'rmagick', '2.16.0'
+#sinatra
 gem 'sinatra', '1.4.7'
+gem 'sinatra-contrib'
```

Comment:
Por qué necesitas de ésta gema? Segun la página de repo, ésto fue merged con la gema sinatra: `NOTE: This project has been merged upstream to sinatra/sinatra https://github.com/sinatra/sinatra/tr…`

Ref: [https://github.com/sinatra/sinatra-contrib](https://github.com/sinatra/sinatra-contrib)